tests/api_test.py

```diff
@@ -379,6 +381,68 @@ def test_update_newdle(flask_client, dummy_newdle, dummy_uid):
     assert resp.json == expected_json
 
 
+@pytest.mark.usefixtures('dummy_newdle')
+def test_update_newdle_changes_last_update(flask_client, dummy_uid, dummy_newdle):
+    final_dt = '2019-09-12T13:30'
```

Comment:
It's used only once so I'd just define it directly in the JSON data below

Reply:
Changed.